docs/source/README.md

```diff
@@ -169,7 +169,7 @@ Source parameters
 |--------|--------|
 | `rabbitmqClusterReference` | A reference to a RabbitMQ Cluster |
 | `rabbitmqClusterReference.name`| This is the name of the RabbitMQ Cluster CRD, if this is set then the `connectionSecret` must not be set |
-| `rabbitmqClusterReference.connectionSecret.name`| The RabbitMQ Connection Secret name, if this is set then the `clusterReference.name` must not be set |
+| `rabbitmqClusterReference.connectionSecret.name`| The RabbitMQ Connection Secret name. The secret must contain `uri`, `username`, `password`, `port* (default 5672)` (* optional). If this is set then the `clusterReference.name` must not be set |
```

Comment:
The secret must contain `uri`, `username` and `password`. It may optionally contain `port` (default 5672).